packages/devextreme/scss/widgets/fluent/htmlEditor/_index.scss

```diff
@@ -273,6 +279,10 @@
   }
 }
 
+.dx-formdialog .dx-formdialog-field-target .dx-field-item-content {
```

Comment:
There is rule in base/htmleditor.scss 
```
.dx-formdialog-field-target .dx-field-item-content {
  text-align: end;
}
```
So, it will be enough to use two selectors and we may don't use additional `.dx-form-dialog` selector for overriding the `text-align` property.

Reply:
[54f050216879d6bb4a1219a7e04ed8976cb02a15]